### ISSUE TITLE
Add support for max_submission_tokens to presets

### ISSUE THREAD

**Issue by joshcho** [llm-workflow-engine/llm-workflow-engine#322]
**Describe the bug**
In the config, unclear as to how to set max_submission_tokens (in the preset or in the config?). Also, preset-save and -load do not operate correctly with max_submission_tokens. Possible that max_submission_tokens is not associated with model preset, but then this is not documented.

**To Reproduce**
1)
Set config file for the preset:

```
metadata:
  name: gpt-4
  provider: chat_openai
model_customizations:
  max_submission_tokens: 2000
  model_name: gpt-4
  n: 1
  request_timeout: null
  streaming: true
  temperature: 1.0
```

Says ```Invalid key```.

2)
1. Set max-submission-tokens via
`/max-submission-tokens 2000`
2. Save preset via `/preset-save my-preset`
3. Load preset via `/preset-load my-preset`

**Expected behavior**
It should work as expected, loading the customization.

**Comment by thehunmonkgroup** [comment]
`max-submission-tokens` is not currently supported in preset configuration, it is a runtime only setting.

I would look at a PR for this feature if it was well considered.

**Comment by joshcho** [comment]
I think even in runtime, max-submission-tokens doesn't quite work. I am getting `KeyError('content')` when my token count goes above max-submission-tokens.

**Comment by thehunmonkgroup** [comment]
> I think even in runtime, max-submission-tokens doesn't quite work

Yep, good catch. Fixed in the 0.14.1 release: https://github.com/llm-workflow-engine/llm-workflow-engine/releases/tag/v0.14.1
